config.schema.json

```diff
@@ -154,6 +154,12 @@
                           "smoke"
                         ]
                       },
+                      {
+                        "title": "Carbon Monoxide Sensor",
+                        "enum": [
+                          "carbon"
```

Comment:
Please keep the alphabetical sort order by type.  Also in the other lists and descriptions.